assets/js/push.js

```diff
@@ -237,11 +237,16 @@ jQuery( window ).on( 'load', () => {
 	 * Handle exiting the distributor menu.
 	 */
 	function distributorMenuExited() {
+
+		if ( actionWrapper && actionWrapper.classList.contains( 'loading' ) ) {
```

Comment:
@dinhtungdu This does get rid of the JS error, which is great. But in my testing, if hover over the Distributor menu and then quickly hover out, while connections are still be loaded in, the menu closes (because the `actionWrapper` doesn't exist yet, so this check is bypassed).

I think we may want to change this to:
```
if ( ! actionWrapper || actionWrapper.classList.contains( 'loading' ) )
```

Reply:
I made it that way because if I hover over the menu then quickly hover out, I consider it as accident behavior. But after testing again, closing the menu after quickly hovering out looks buggy.
I added a check to cover the case when we have connections loading issue as well.